docs/src/architecture/state.md

```diff
@@ -30,27 +30,27 @@ As described in [accounts](https://0xpolygonmiden.github.io/miden-base/architect
 
 > Losing a state of a private account would mean loss of funds (as the user won't be able to execute transactions) in a similar manner as a loss of a private key would. This problem can be easily mitigated by storing encrypted account state in a cloud or backing it up somewhere else. Unlike storing private keys in the cloud, this does not compromise privacy or security of an account.
 
-Note: Having many (or even most) of the accounts be private is very beneficial for the network as a private account contributes only 64 bytes to the global state (32 bytes account ID + 32 bytes account hash). Or, said another way, 1 billion private accounts takes up only $60$ GB of state.
+Note: Having many (or even most) of the accounts be private is very beneficial for the network as a private account contributes only 40 bytes to the global state (8 bytes account ID + 32 bytes account hash). Or, said another way, 1 billion private accounts takes up only $40$ GB of state.
```

Comment:
This is a bit oversimplified (this is the ideal number, in reality the data structures required to store this take up a bit more space) - but I think it is fine to have it here as an illustrative point.

Reply:
Thanks for the clarification